Plugins/Public/base_plugin/PlayerCommands.cpp

```diff
@@ -1736,6 +1742,65 @@ namespace PlayerCommands
 		pub::Player::PopUpDialog(client, caption, message, POPUPDIALOG_BUTTONS_CENTER_OK);
 	}
 
+	//FL only supports one "price" in the dealer menu, this function gives players a command to 

```

Comment:
What happens when the player buys item, when the view shows not the right view? Have you tried doing it?

Reply:
The dealer view will not always show the correct price without use of the /price command but will handle the transaction correctly. There is no way I can find that make FL change the displayed value when only clicked upon. So if a player is making a purchase and use /price sell the view will show correct values.

Reply:
I would imagine it would add some confusion to players.
Imagine the player visits the base and it has Buy prices mod. He happily starts to buy for the price he thinks the base is selling one.
And he discovers it is much higher than the buying price. Perhaps to give him in some way some warning at least?
Or may be even canceling the purchase if he is not in the right mod?

Reply:
You raise a good point, I put a contingency in place: I changed the market display  logic (such that I could) so that when you first enter the market it displays the bases selling prices, unless you open the market with something on hand that the base has 0 of, in which case it displays what the base buys it for